firmware/port.c

```diff
@@ -84,6 +84,8 @@ void port_init(PortData* p, u8 chan, const TesselPort* port,
 
     sercom_clock_enable(p->port->spi, p->clock_channel, 1);
     sercom_clock_enable(p->port->uart_i2c, p->clock_channel, 1);
+
+    bridge_enable_chan(chan);
```

Comment:
I think instead of enabling the channel from the MCU, we should enable it from `tessel-export.js` when a Port object is initialized. Otherwise, the port LEDs come on immediately.